runtime/src/storage/audited.rs

```diff
@@ -101,14 +101,6 @@ impl<B: crate::Blob> crate::Blob for Blob<B> {
         });
         self.inner.sync().await
     }
-
-    async fn close(self) -> Result<(), Error> {
```

Comment:
I think removing the `close` event is fine -- we still have the auditor event for sync.